libc/src/__support/FPUtil/BasicOperations.h

```diff
@@ -73,6 +75,20 @@ LIBC_INLINE T fdim(T x, T y) {
   return (x > y ? x - y : 0);
 }
 
+template <typename T, cpp::enable_if_t<cpp::is_floating_point_v<T>, int> = 0>
+LIBC_INLINE T canonicalize(T *cx, const T *x) {
```

Comment:
Change the parameter types to `T& c` and `const T &x`.  Update the call sites accordingly.

Reply:
done.